gc/gc-tool/src/main/java/org/projectnessie/gc/tool/cli/options/EnvironmentDefaultProvider.java

```diff
@@ -38,7 +39,7 @@ public String defaultValue(ArgSpec argSpec) {
       OptionSpec optionSpec = (OptionSpec) argSpec;
       return Arrays.stream(optionSpec.names())
           .filter(name -> name.startsWith("--"))
-          .map(name -> "NESSIE_GC_" + name.substring(2).replace('-', '_').toUpperCase())
+          .map(name -> "NESSIE_GC_" + name.substring(2).replace('-', '_').toUpperCase(Locale.ROOT))
```

Comment:
should we set the new upper/lowercase locale check to ERROR ?

Reply:
Mind opening a PR?